client/components/gravatar/index.jsx

```diff
@@ -64,10 +73,16 @@ module.exports = React.createClass( {
 		}
 
 		const alt = this.props.alt || this.props.user.display_name;
-		const avatarURL = this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
+
+		const avatarURL = this.props.tempImage ||
+			this.getResizedImageURL( safeImageURL( this.props.user.avatar_URL ) );
```

Comment:
Looks like you added some more detail to your comment since I replied to it :). Given the fact that using booleans to get a value is a well-known pattern , I don't see a big advantage to adding a helper function here.